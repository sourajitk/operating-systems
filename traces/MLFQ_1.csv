1,0,15
2,4,4
3,6,1
4,8,4
5,10,2
6,11,10
