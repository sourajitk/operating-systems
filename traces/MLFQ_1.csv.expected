1,36
2,17
3,7
4,18
5,13
6,31